frontend/src/modules/editor/actions.js

```diff
@@ -147,10 +149,15 @@ export function sendTranslation(
                     content.translation
                 )
             );
-
-            if (nextEntity && nextEntity.pk !== entity) {
-                // The change did work, we want to move on to the next Entity.
-                dispatch(navActions.updateEntity(router, nextEntity.pk.toString()));
+            if (nextEntity) {
+                // The change did work, we want to move on to the next Entity or pluralForm.
+                const nextPluralForm = pluralForm + 1;
+                if (pluralForm !== -1 && nextPluralForm < locale.cldrPlurals.length) {
+                    dispatch(pluralActions.select(nextPluralForm));
+                }
+                else if (nextEntity.pk !== entity) {
+                    dispatch(navActions.updateEntity(router, nextEntity.pk.toString()));
+                }
```

Comment:
This code is repeated in `frontend/src/modules/history/actions.js`. I'd like to have it on once place. Where should that be? Maybe `navActions.updateEntityOrPlural`?